app/uk/gov/hmrc/minorentityidentificationfrontend/models/OverseasCompanyDetails.scala

```diff
@@ -62,4 +62,32 @@ object OverseasCompanyDetails {
       }
     }
   }
+
+  def writesForAudit(optOverseasCompanyDetails: Option[OverseasCompanyDetails]): JsObject = {
+    optOverseasCompanyDetails match {
+      case Some(overseasDetails) =>
+        val optUtrBlock = overseasDetails.optUtr match {
+          case Some(utr: Ctutr) => Json.obj("userCTUTR" -> utr.value, "isMatch" -> "false")
```

Comment:
It looks like the "isMatch" property is set along with the utr value. However, this means that if the user doesn't define a Utr "isMatch" is not defined.

Reply:
Updated it, just kept it as false as I imagine there is a story to update the Overseas Company auditing so I don't want to change too many of the value in this story